tendrl/node_agent/manager/manager.py

```diff
@@ -101,25 +102,42 @@ def __init__(self, machine_id):
         self.etcd_client = etcd.Client(**etcd_kwargs)
 
         self.register_node(machine_id)
+        self._alerts_manager = AlertsManager()
 
     def stop(self):
         LOG.info("%s stopping" % self.__class__.__name__)
         self._user_request_thread.stop()
         self._discovery_thread.stop()
         self.persister.stop()
+        self._alerts_manager.stop()
 
     def start(self):
         LOG.info("%s starting" % self.__class__.__name__)
         self._user_request_thread.start()
         self._discovery_thread.start()
         self.persister.start()
+        self._alerts_manager.start()
 
     def join(self):
         LOG.info("%s joining" % self.__class__.__name__)
         self._user_request_thread.join()
         self._discovery_thread.join()
         self.persister.join()
 
+    def append_definitions(self):
+        defs_path = 'tendrl_definitions_node_agent/data'
```

Comment:
@r0h4n I'll probably change this once the framework changes related to this are in because otherwise, I won't be able to dev-test this code